Fix System.run() documentation in minecraft server System.md

glinto opened the pull request:
- run() method description is wrong (describes runTimeout)
- example will not display message in every minute as the script suggests, unless we wrap it in a function and re-run every tick

glinto commented:
@microsoft-github-policy-service agree

mammerla commented:
@glinto I think we've fixed the example subsequently so I'm closing the PR on this basis, but I have incorporated your fix for system.run into our API documentation. Thanks for opening up this PR! (and sorry it took so long for me to get to it.)

glinto commented:
> @glinto I think we've fixed the example subsequently so I'm closing the PR on this basis, but I have incorporated your fix for system.run into our API documentation. Thanks for opening up this PR! (and sorry it took so long for me to get to it.)

No worries @mammerla I'm glad it got fixed eventually and that I could contribute
